src/features/favorites/pages/Favorites.native.test.tsx

```diff
@@ -31,9 +31,10 @@ describe('<Favorites/>', () => {
 
   it('should render correctly', async () => {
     renderFavorites({ isLoggedIn: true })
-    await act(async () => {})
 
-    expect(screen).toMatchSnapshot()
+    await waitFor(() => {
+      expect(screen).toMatchSnapshot()
+    })
   })
```

Comment:
sur les snapshots je trouve que ça a pas trop de sens non plus... Un waitFor... to matchSnapshot c'est pas vraiment ce qu'on veut tester => on veut qu'à la fin de tous les rendus, ça match, et dans ce cas là je garderais bien le act...

Reply:
Pour le snapshot je vais plutôt await un text présent dans le snapshot